pkgs/development/libraries/SDL_mixer/default.nix

```diff
@@ -1,4 +1,9 @@
-{ stdenv, lib, fetchurl, SDL, libogg, libvorbis, smpeg, enableNativeMidi ? false, fluidsynth ? null }:
+{ stdenv, lib, fetchurl
+, SDL, libogg, libvorbis, smpeg, libmikmod
+, enableNativeMidi ? false
+, fluidsynth ? null
```

Comment:
```suggestion
, fluidsynth
```

Reply:
Can you explain? Wouldn't this add fluidsynth as an unconditional dependency?

Reply:
``fluidsynth`` is always defined in all-packages.nix and if you overwrite it with null this does nothing because it is already null.

Reply:
Aha, I see. Thank you for your patience :-)